### PR TITLE
Lots of things

### PR DESCRIPTION
**Pull request by fiji-flo**
- use graphql data in user menu
- fix view in orgchart (scrolling and expanding to person)
- use graphql for access groups
- use structured data for ssh/pgp keys

We copy the `user-demo.png` in the Dockerfile since it's no longer considered an asset by Vue. We should fix that.

### CODE REVIEW

File: src/components/OrgNode.vue [hunk 1]
```diff
@@ -44,11 +43,6 @@ export default {
       }
     }
   },
-  computed: {
```
Comment (hidde) [inline src/components/OrgNode.vue]: I suspect something happened to replace this? Can't see what it is, but if there is a replacement, I'm good with that! 🎉

File: src/components/Profile.vue [hunk 2]
```diff
@@ -177,13 +177,14 @@ export default {
     description: Object,
     created: Object,
     lastModified: Object,
-    pgpPublicKeys: Object,
-    sshPublicKeys: Object,
+    pgpPublicKeys: Array,
```
Comment (hidde) [inline src/components/Profile.vue]: Aaah… that's what I missed when I tried to fix this last night 😄

File: src/components/TopBar.vue [hunk 3]
```diff
@@ -48,80 +67,81 @@ export default {
...
-      .top-bar__user-menu-toggle img {
-        margin-right: 0;
-        border-radius: var(--imageRadius);
-      }
-      .top-bar__user-menu-toggle .show-more__button-text {
-        position: absolute;
-        left: -9999em;
-        top: -9999em;
-      }
+.top-bar__bar {
```
Comment (hidde) [inline src/components/TopBar.vue]: ok ok 😄 If you prefer no indentation, let's change it across the board. I can do that sometime in next few weeks.

File: src/components/functional/ShowMore.vue [hunk 4]
```diff
@@ -49,7 +45,7 @@ export default {
   watch: {
     trace() {
       if (this.trace || this.prefix) {
-        this.expanded = this.prefix === this.trace || this.trace.startsWith(`${this.prefix}-`) || this.expanded;
+        this.expanded = this.trace.startsWith(`${this.prefix}-`) || this.expanded;
```
Comment (hidde) [inline src/components/functional/ShowMore.vue]: Ah, it moved to here! Note that we use `ShowMore` in all places where we show/hide something (I might rename to `ShowHide` or `Toggle` at some point).

File: src/components/TopBar.vue (hunk 3 above)
Comment (fiji-flo) [reply src/components/TopBar.vue]: Sorry, for that.

If you have a strong feeling about that I can try to teach my editor to accept this :P

File: src/components/functional/ShowMore.vue (hunk 4 above)
Comment (fiji-flo) [reply src/components/functional/ShowMore.vue]: Jupp, I'm aware. When I played around with this, everything started to pop out.

**Pull request merged**